Content.Client/Preferences/UI/HumanoidProfileEditor.cs

```diff
@@ -529,14 +605,14 @@ public partial class HumanoidProfileEditor : Control
 
                         foreach (var jobSelector in _jobPriorities)
                         {
-                            // Sync other selectors with the same job in case of multiple department jobs
-                            if (jobSelector.Job == selector.Job)
+                // Sync other selectors with the same job in case of multiple department jobs
+                if (jobSelector.Job == selector.Job)
                             {
                                 jobSelector.Priority = priority;
                             }
 
-                            // Lower any other high priorities to medium.
-                            if (priority == JobPriority.High)
+                // Lower any other high priorities to medium.
+                if (priority == JobPriority.High)
```

Comment:
whitespace moment?

Reply:
yeah, somebody did it like that, auto formatting fixed it)